common/scala/src/main/resources/application.conf

```diff
@@ -172,7 +177,7 @@ whisk {
 
     activation {
         payload {
-            max = 1048576 // 5 m not possible because cross-referenced to kafka configurations
+            max = 1048576 // value is cross-referenced to kafka configurations for whisk.kafka.payload.max
```

Comment:
can we avoid having this value defined twice? here and as `limit_activation_payload: 1048576 # Maximum action invocation payload and response size` in the environment?